docs/source/markdown/podman-commit.1.md

```diff
@@ -36,6 +36,11 @@ Apply the following possible instructions to the created image:
 
 Can be set multiple times.
 
+#### **--config**=*ConfigBlobFile*
+
+Merge the JSON-encoded container configuration from the specified file into the configuration for the image
```

Comment:
Can we point users to a specification of the configuration?

Reply:
It varies between OCI and Docker image formats.  I don't think there's a stable location to point to for the Docker config blob format (the engine API documentation is close, but it includes the API version number), and the OCI version is missing some interesting fields.  If it wasn't being exercised in tests that I'm about to add, I could pretty easily be talked into dropping the option.
Actually, maybe even after I add the tests to the PR....

Reply:
Well, as written, it's a JSON-encoded copy of the structure defined at https://github.com/containers/image/blob/main/manifest/docker_schema2.go#L67.  Would a pointer to that be useful enough to someone who hasn't spent time staring at that code at some point already?

Reply:
Great idea, thanks @nalind

Reply:
Okay, added.  Pointed at the currently-newest tag instead of `main` in case the line number changes.